mllib/src/main/scala/org/apache/spark/ml/functions.scala

```diff
@@ -60,6 +58,15 @@ object functions {
    *
```

Comment:
Add parameter and return type description in doc.
say the return type include two case: `array<float>` or `array<double>`